CMakeLists.txt

```diff
@@ -170,8 +170,7 @@ configure_file(libnaboConfig.cmake.in
 # 2- installation build #
 
 # Change the library location for an install location
-get_filename_component(NABO_LIB_NAME ${libnabo_library} NAME)
-set(libnabo_library ${CMAKE_INSTALL_PREFIX}/lib/${NABO_LIB_NAME})
+set(libnabo_library ${LIB_NAME})
```

Comment:
I looked at my diffs from March/April but I can't seem to remember what that was for. My guess is that it helped Arch Linux compatibility. 

I did not have the installation in a catkin workspace in my test cases. 

My Arch machine is at home and i'll try this out tonight. I guess it's best to revert the changes in the meantime.